### PR TITLE
added style to webpage

### PR DESCRIPTION
**Pull request by Outkarm**
in this pull request styling was added to page

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -34,28 +34,30 @@ const Input = () => {
...
-          onChange={addNewBook}
-        />
+      <h2 className="add-book">Add books</h2>
+      <div className="imput-cont">
+        <div className="title">
+          <input
+            type="text"
+            id="text"
+            placeholder="Book Title"
+            // value={booksState.title}
```
Comment (AmaduKamara) [inline src/components/Form.js]: - [ ] For codes that you're not using in your project, it's best to delete or remove them so your codebase looks clean and professional. It's a good practice to not have commented codes in our codebase and that makes your project clean and professional. Kindly remove this 👍🏽

File: src/components/Form.js [hunk 2]
```diff
@@ -34,28 +34,30 @@ const Input = () => {
...
+            onChange={addNewBook}
+          />
+        </div>
+        <div className="author">
+          <input
+            type="text"
+            id="text-author"
+            placeholder="Book Author"
+            // value={booksState.author}
```
Comment (AmaduKamara) [inline src/components/Form.js]: - [ ] For codes that you're not using in your project, it's best to delete or remove them so your codebase looks clean and professional. It's a good practice to not have commented codes in our codebase and that makes your project clean and professional. Kindly remove this 👍🏽

File: src/styles/App.css [hunk 3]
```diff
@@ -1,5 +1,80 @@
...
+  letter-spacing: -0.15px;
+  color: #121212;
+}
+
+.add-book {
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.18px;
+  color: #888;
+}
```
Comment (AmaduKamara) [inline src/styles/App.css]: - [ ] The input field, select option for author and the button for adding book do not match the original design styles. The styles for your input fields and the button does not look nice or as close to the original design given and it's a requirement for your styles to match the given design. Kindly style the input field for the book title, author select option, and button so they look like the template design given. 👍🏽

Screenshot 2023-02-04 at 8 23 43 PM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @Outkarm,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] When the app loads, there are no books showing on the page and I can't tell if the styles for single books match the design given. Kindly work on making the books show on the page and styled as per the design given. 👍🏽

Screenshot 2023-02-04 at 8 26 34 PM

- [ ] When I click on the route for categories page, I noticed that the route path on the navigation link is different from the Route path set in your routing. Kindly visit the App.js and adjust the route paths for the navigation Link and the Route set in the react-router Route component for categories so they match and then the actual page is rendered when that route is clicked 👍🏽

Screenshot 2023-02-04 at 8 22 35 PM

- [ ] Kindly style your input fields to match the original design and also, when you work on displaying the books, please try to have your design match the design in the screenshot below. Kindly take note of the user icon in the navigation bar, it's missing in your design at the moment. Please work on them 👍🏽

Screenshot 2023-02-04 at 8 26 34 PM


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mikethreels — APPROVED** [review]
Hi Outkarm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**